lib/blockchain/chain.js

```diff
@@ -550,6 +550,24 @@ class Chain extends AsyncEmitter {
             100);
         }
 
+        if (this.height >= this.network.goosigStop) {
+          const key = proof.getKey();
+
+          if (!key) {
+            throw new VerifyError(block,
+              'invalid',
+              'bad-airdrop-format',
```

Comment:
nit: should this reason read 'bad-airdrop-proof'?

Reply:
I used `bad-airdrop-format` since the same error message is being used earlier in the validation.

https://github.com/handshake-org/hsd/blob/e2039e895484b17a43ac025a87a033aed2fe20ba/lib/blockchain/chain.js#L540-L544

Reply:
Updated since `bad-xxx-format` is used when decoding specifically